keras/backend/cntk_backend.py

```diff
@@ -703,7 +725,17 @@ def squeeze(x, axis):
         del shape[_]
 
     new_shape = shape[nones:]
-    new_shape = tuple([C.InferredDimension if _ == C.FreeDimension else _ for _ in new_shape])
+
+    new_shape_temp = []
+    for _ in new_shape:
+        if _ == C.FreeDimension:
+            new_shape_temp.append(C.InferredDimension)
+        else:
+            new_shape_temp.append(_)
+
+    new_shape = tuple(new_shape_temp)
+    del new_shape_temp
```

Comment:
`new_shape_temp` will be deleted automatically after the return statement. There is no need to delete it explicitly.

Reply:
removed del command from the code.
kindly review.